usecase/usecase-wrt-linux-tests/res/testapp/org.csp.cspdefaultsrc/index.html

```diff
@@ -25,22 +26,31 @@
 EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 Authors:
-        Hao, Yunfei <yunfeix.hao@intel.com>
+        Xu, Kang <kangx.xu@intel.com>
 
-*/
+-->
 
-#test-blue {
-    background-color: blue;
-    height: 100px;
-    width: 100px;
-}
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>CSP Test: default-src_self</title>
+    <style>
+      #test1 {
+        width: 100px;
+        height: 100px;
+        background: url("./image/cat.png") no-repeat;
+      }
 
-.button {
-    background: #eee;
-    border: 1px solid #ccc;
-    border-radius: 5px;
-    font-weight: bold;
-    font-size: 15px;
-    padding: 8px 2px;
-    width: 100%;
-}
+      #test2 {
+        width: 100px;
+        height: 100px;
+        background: url("https://hg.csswg.org/test/raw-file/b90524bf2d15/support/cat.png") no-repeat;
```

Comment:
No, we don't want this dependency. This image may have no effect if there is a networking connection.